agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -251,53 +251,6 @@ bool slave_thread::work()
     return true;
 }
 
-void slave_thread::process_keep_alive()
-{
-    if (!config.enable_keep_alive || !son_config.slave_keep_alive_retries ||
```

Comment:
I think `enable_keep_alive` and `slave_keep_alive_retries` configuration attributes are not used any more and should be removed too

Reply:
fixed